testdata/project-v4-multigroup-with-deploy-image/cmd/main.go

```diff
@@ -78,11 +78,14 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
-	var probeAddr string
-	var secureMetrics bool
-	var enableHTTP2 bool
+	var (
+		enableLeaderElection bool
+		secureMetrics        bool
+		enableHTTP2          bool
+		metricsAddr          string
+		probeAddr            string
+		tlsOpts              []func(*tls.Config)
+	)
```

Comment:
Can we please keep the var in each line as before ? 
So that it make easier for users just comment and etc if they wish?

Reply:
![0cf1af17e53154e68fe7f5105a21566c](https://github.com/user-attachments/assets/cd87e000-0796-43ef-9474-366af77d57b8)
Is it in the order of variables, so that the hierarchy is clearer?

Reply:
If you change it to var variable, the meaning of var code block will be lost.

Reply:
No, I mean change the bullerplate to keep all as it was before and just add your change on it.

```
var metricsAddr string
var enableLeaderElection bool
var probeAddr string
var secureMetrics bool
var enableHTTP2 bool
```
^ has no value on change those


```
var metricsAddr string
var enableLeaderElection bool
var probeAddr string
var secureMetrics bool
var enableHTTP2 bool
tlsOpts              []func(*tls.Config) <- HERE
```